Dockerfile

```diff
@@ -50,7 +56,9 @@ COPY --from=backend-builder /opt/app/package.json ./
 WORKDIR /opt/app
 
 EXPOSE 3000
+
+# Add a health check to ensure the container is healthy
 HEALTHCHECK --interval=10s --timeout=3s CMD curl -f http://localhost:3000/api/health || exit 1
 
-# HOSTNAME=0.0.0.0 fixes https://github.com/vercel/next.js/issues/51684. It can be removed as soon as the issue is fixed
-CMD cp -rn /tmp/img /opt/app/frontend/public && HOSTNAME=0.0.0.0 node frontend/server.js & cd backend && npm run prod
+# Application startup
```

Comment:
```suggestion
# Application startup
# HOSTNAME=0.0.0.0 fixes https://github.com/vercel/next.js/issues/51684. It can be removed as soon as the issue is fixed
```

Reply:
Sure, added the comment back.